apps/api/src/donations/donations.controller.ts

```diff
@@ -94,6 +94,12 @@ export class DonationsController {
     return await this.donationsService.getDonationsByUser(user.sub)
   }
 
+  @Get('users-donated')
+  @Public()
+  async usersDonated() {
```

Comment:
[naming] in this way the endpoint will be /api/v1/donation/user-donations, however it actually returs user-count, so let's name it like this: /api/v1/donation/donors-count